.github/ISSUE_TEMPLATE/bug-core.md

```diff
@@ -1,13 +1,11 @@
 ---
+
```

Comment:
Could you please update this PR to not be applying formatting changes to the .github templates?